subprojects/android-test/test-app/build.gradle.kts

```diff
@@ -90,6 +90,9 @@ dependencies {
     androidTestImplementation(project(":subprojects:common:okhttp"))
     androidTestImplementation(project(":subprojects:common:time"))
 
+    androidTestImplementation(Dependencies.androidTest.runner)
+    androidTestUtil(Dependencies.androidTest.orchestrator)
```

Comment:
What is an androidTestUtil?

Reply:
It's an [official way](https://developer.android.com/training/testing/junit-runner#ato-gradle) to use runner.